commcare_connect/opportunity/models.py

```diff
@@ -70,3 +70,70 @@ class DeliverForm(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class LearnModule(models.Model):
+    app = models.ForeignKey(
+        CommCareApp,
+        on_delete=models.CASCADE,
+        related_name="learn_module",
+        related_query_name="learn_module",
+    )
+    slug = models.SlugField()
+    name = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
+    time_estimate = models.IntegerField()
+
+
+class UserModule(models.Model):
```

Comment:
how about `CompletedModule`?
```suggestion
class CompletedModule(models.Model):
```